pkg/registry/authentication/tokenreview/storage.go

```diff
@@ -44,6 +45,7 @@ func (r *REST) Create(ctx genericapirequest.Context, obj runtime.Object, include
 	if !ok {
 		return nil, apierrors.NewBadRequest(fmt.Sprintf("not a TokenReview: %#v", obj))
 	}
+	validation.ValidateTokenReview(tokenReview)
```

Comment:
@liggitt Really glad to hear your feedback!

> you're calling validation but not doing anything with the result

I will fix it. Thanks for reminding.

> I'd probably inline the length check on the spec.token rather than adding a whole package just to check that one field.

Yes, but I think creating a validation package for TokenReview is more consistent - I find TokenReview is the only resource which misses validation package in `pkg/apis`.

Besides, create a validation package can help people doing more checks on TokenReview in the future and add some UTs.